dashboard/src/utils/utils.ts

```diff
@@ -94,6 +101,21 @@ export const sanitizeBuilds = (
   }));
 };
 
+export const sanitizeBuildTable = (
```

Comment:
This is creating an O(2N), and sometimes N equals 600000, maybe it is better to create an object with the sanitizers and place them on the row definition, but the only way of knowing is if we measure.

Reply:
The 600000 figure is for tests related to an issue, for builds the number is much lower. But I agree we should fix the typing of builds so that we won't need all this conversion.